From 40d425fb2b82c96388e38fb0bce0329f3c5fdd05 Mon Sep 17 00:00:00 2001
From: Razvan Stefanescu <razvan.stefanescu@nxp.com>
Date: Fri, 20 Oct 2017 16:14:38 +0300
Subject: [PATCH 168/767] staging: dpaa2-evb: Update
 netdev_master_upper_dev_link parameters

Signed-off-by: Razvan Stefanescu <razvan.stefanescu@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/staging/fsl-dpaa2/evb/evb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/fsl-dpaa2/evb/evb.c b/drivers/staging/fsl-dpaa2/evb/evb.c
index 23d32dad73df..3f0782f5a7fa 100644
--- a/drivers/staging/fsl-dpaa2/evb/evb.c
+++ b/drivers/staging/fsl-dpaa2/evb/evb.c
@@ -1268,7 +1268,7 @@ static int evb_probe(struct fsl_mc_device *evb_dev)
 
 			rtnl_lock();
 			err = netdev_master_upper_dev_link(port_netdev, netdev,
-							   NULL, NULL);
+							   NULL, NULL, NULL);
 			if (unlikely(err)) {
 				dev_err(dev, "netdev_master_upper_dev_link err %d\n",
 					err);
-- 
2.17.0

